kong/router/traditional.lua

```diff
@@ -1448,6 +1450,21 @@ function _M.new(routes, cache, cache_neg)
   local match_sources        = not isempty(plain_indexes.sources)
   local match_destinations   = not isempty(plain_indexes.destinations)
 
+  -- warning about the regex cache size being too small
+  if not lua_regex_cache_max_entries then
+    lua_regex_cache_max_entries = kong.configuration.nginx_http_lua_regex_cache_max_entries
```

Comment:
`tonumber(kong.configuration.nginx_http_lua_regex_cache_max_entries) or 1024` is OK,
Because `tonumber(nil) == nil` `tonumber('xxx') == nil`